components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -110,7 +111,74 @@
 
  @param token The suspension token to invalidate.
  */
-+ (void)resumeMessagesWithToken:(id<MDCSnackbarSuspensionToken>)token;
++ (void)resumeMessagesWithToken:(nullable id <MDCSnackbarSuspensionToken>)token;
+
+#pragma mark Styling
+
+/**
+ The color for the background of the snackbar message view.
+ */
+@property(class, nonatomic, strong, nullable) UIColor *snackbarMessageViewBackgroundColor;
+
+/**
+ The color for the shadow color for the snackbar message view.
+ */
+@property(class, nonatomic, strong, nullable) UIColor *snackbarMessageViewShadowColor;
+
+/**
+ The color for the message text in the snackbar message view.
+ */
+@property(class, nonatomic, strong, nullable) UIColor *messageTextColor;
+
+/**
+ The font for the message text in the snackbar message view.
+ */
+@property(class, nonatomic, strong, nullable) UIFont *messageFont;
+
+/**
+ The font for the button text in the snackbar message view.
+ */
+@property(class, nonatomic, strong, nullable) UIFont *buttonFont;
+
+
+/**
+ When setting this BOOL to YES, then all the styling changes that are set using the class styling
```

Comment:
minor grammar nit: "If enabled, modifications of class styling properties will be applied immediately to the currently presented snackbar."

Reply:
done